src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceDeploy.kt

```diff
@@ -34,26 +34,25 @@ open class InstanceDeploy : Instance() {
     fun deploy() {
         instanceManager.examine(anyInstances)
```

Comment:
I have updated in the meantime this task, please ensure that it is running awaitUp as in master
https://github.com/wttech/gradle-aem-plugin/blob/master/src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceDeploy.kt#L45